prime-router/build.gradle.kts

```diff
@@ -118,6 +118,7 @@ val env = mutableMapOf<String, Any>(
     "AzureWebJobsStorage" to devAzureConnectString,
     "AzureBlobDownloadRetryCount" to 5,
     "PartnerStorage" to devAzureConnectString,
+    "SftpStorage" to devAzureConnectString,
```

Comment:
Any reason we are not using standard env variable conventions here (all caps with underscores)?

Reply:
I think this is an azure thing `AzureWebJobsStorage` is actually in there docs everywhere